### DISCUSSION TITLE
purge msql values

### DISCUSSION THREAD

**Discussion by RossWilliamson** [SBFspot/SBFspot#484] [General]
My mariadb/mysql database is pretty large and it slows down considerably.  Do you have a good command to purge all data that is older than 3 months?

**Comment by mincequi** [comment]
"Retention policy" is the key word. Well, this gets a little off-topic now, but i want to throw in _InfluxDB_, since you will get this feature for free using it. SBFspot should support this as a DB backend (with the possibility to import historical data from other DBs).
I am already experimenting with it, but i did not find the time to integrate this into SBFspot.

**Comment by sbf-** [comment]
To keep the last 180 days: (use select statement first to check)
``` SQL
SELECT * FROM daydata
WHERE from_unixtime(TimeStamp) < NOW() - INTERVAL 180 DAY;
```
When query is OK, delete the records:
``` SQL
DELETE FROM daydata
WHERE from_unixtime(TimeStamp) < NOW() - INTERVAL 180 DAY;
```
Do the same for spotdata table
